lib/pipeline.ts

```diff
@@ -82,6 +83,16 @@ export interface PipelineProps {
    * @default true
    */
   restartExecutionOnUpdate?: boolean;
+
+  /**
+   * Indicates the concurrency limit test and publish stages.
+   *
+   * For example, if this value is 2, then only two actions will execute concurrently.
+   * If this value is 1, the pipeline will not have any concurrent execution.
+   *
+   * @default no limit
```

Comment:
The reason I had mentioned a default of 10 is because it used to be the CodePipeline limit for parallel actions in a stage. But in fact, this was increased to 50 recently. I would tend to default to 50 for that reason, but I'm also happy with "no limit".